std/range/primitives.d

```diff
@@ -1638,6 +1638,7 @@ upTo) steps have been taken and returns $(D upTo).
 Infinite ranges are compatible, provided the parameter $(D upTo) is
 specified, in which case the implementation simply returns upTo.
  */
+deprecated("use std.algorithm.searching.count instead.")
```

Comment:
Also, that deprecated string is for symbols that have already been through a year of deprecation and are now undocumented. So, even if we deprecated `walkLength` (which IMHO, we shouldn't), this would not be the message to use. It would have to be one from a symbol that's still documented with a ddoc comment.